lib/slim_fast/parser.ex

```diff
@@ -95,6 +75,39 @@ defmodule SlimFast.Parser do
     end
   end
 
+  defp parse_attributes(""), do: {"", []}
+  defp parse_attributes(nil), do: {"", []}
+  defp parse_attributes("(" <> line), do: line |> unwrap_attributes |> parse_attributes
+  defp parse_attributes("[" <> line), do: line |> unwrap_attributes |> parse_attributes
+  defp parse_attributes("{" <> line), do: line |> unwrap_attributes |> parse_attributes
+
+  defp parse_attributes(line, acc \\ []) do
+    line = String.strip(line)
+    case Regex.run(@attr_regex, line) do
+      [token, key, value] ->
+        acc = html_attribute(key, value) ++ acc
+        index = String.length(token) + 1
+        {_, line} = line |> String.split_at(index)
+
+        parse_attributes(line, acc)
+      nil -> {line, acc}
+    end
+  end
+
+  defp unwrap_attributes(line) do
+    line = case Regex.run(@boolean_regex, line) do
+             [_|[match|_]] ->
+               match = String.strip(match)
+               String.replace(line, match, match <> "=true")
```

Comment:
This would replace all inclusions of matched string, so the following assert fails
```slim
assert render(~s(div [ab="ab" a] a)) == ~s(<div ab="ab" a>a</span>)
```